authors/apps/articles/models.py

```diff
@@ -47,6 +47,16 @@ def save(self, *args, **kwargs):
                                 uuid.uuid4().hex[:6])
         super().save(*args, **kwargs)
 
+    @property
+    def get_time_to_read(self):
+        # Set the standard read time
+        words_per_min = 270
```

Comment:
@pngg Where is the time estimate for words per minute derived from

Reply:
@PAbishai some googling :) and a bit of this https://www.jstor.org/stable/1161646. I intend to move this setting to ENV for easy manipulation/change.